torch/optim/lr_scheduler.py

```diff
@@ -276,8 +273,9 @@ def get_lr(self):
 
 
 class MultiplicativeLR(LRScheduler):
-    """Multiply the learning rate of each parameter group by the factor given
-    in the specified function. When last_epoch=-1, sets initial lr as lr.
+    """Multiply the learning rate of each parameter group by the factor given in the specified function.
+
+    When last_epoch=-1, sets initial lr as lr.
```

Comment:
```suggestion
    When last_epoch=-1, set the initial lr as lr.
```

Reply:
I didn't touch them since they are in a class and not the summary line.